### PR TITLE
Migrate existing API endpoints to this package

### PR DESCRIPTION
**Pull request by pverscha**
This PR migrates all the existing API endpoints from the complete Unipept package to this new Unipept API package.

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+#
+# Table name: posts
+#
+#  id      :integer          unsigned, not null, primary key
+#  title   :string(100)      not null
+#  content :text(65535)      not null
+#  date    :date             not null
+#
+
+class Post < ApplicationRecord
```
Comment (tibvdm) [inline app/models/post.rb:11]: Do we keep the Post models?

File: app/views/mpa/analyze.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<% content_for :javascripts do %>
```
Comment (tibvdm) [inline app/views/mpa/analyze.html.erb:1]: Not sure if this is still needed in the backend

File: test/fixtures/posts.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# == Schema Information
```
Comment (tibvdm) [inline test/fixtures/posts.yml:1]: Fixture for post model. If Post is removed, then this should be removed as well

File: test/models/post_test.rb [hunk 4]
```diff
@@ -0,0 +1,58 @@
+# == Schema Information
```
Comment (tibvdm) [inline test/models/post_test.rb:1]: Post: idem

**Review by tibvdm — APPROVED** [review]

**Comment by pverscha** [comment]
Thanks! @tibvdm I've folllowed your suggestions and removed these files, they were indeed redundant.

**Pull request merged**